LMChatCore_iOS/LMChatCore_iOS/Source/Module/MessageList/LMChatMessageListViewController.swift

```diff
@@ -327,6 +331,21 @@ extension LMChatMessageListViewController: LMMessageListViewModelProtocol {
         messageListView.currentLoggedInUserReplaceTagFormat = viewModel?.loggedInUserReplaceTagValue ?? ""
         messageListView.reloadData()
         bottomMessageBoxView.inputTextView.chatroomId = viewModel?.chatroomViewData?.id ?? ""
+        hideShowTopicBarView()
+    }
+    
+    func hideShowTopicBarView() {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.4) {[weak self] in
+            guard let self else { return }
+            if let firstSection = messageListView.tableSections.first,
+               let message = firstSection.data.first,
+               message.messageType == LMChatMessageListView.chatroomHeader,
+               let _ = messageListView.tableView.cellForRow(at: IndexPath(row: 0, section: 0)) {
```

Comment:
Remove line 343, if we are not using this variable

Reply:
Done